stix_shifter/stix_translation/src/modules/guardium/json/from_stix_map-old.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
This file should be removed, it's not used anywhere.

Reply:
done

Reply:
Removed.